README.md

```diff
@@ -1,14 +1,34 @@
 # kmt-header [![Circle CI](https://circleci.com/gh/Financial-Times/kmt-header.svg?style=svg)](https://circleci.com/gh/Financial-Times/kmt-header)
 
-Header component for KMT app
+Header component for KAT.
 
-[TOC]
+KAT (Knowledge & administration tools) is an ft.com application created for Financial Times B2B clients.
 
-## Building this project
-`npm start` should run a gulp based build before serving up the an example of the page.
-However if you see an error like:
-`Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime`
-You will probably need to run `npm rebuild node-sass` to overcome this. You should probably update `origami-build-tools` at the same time.
+## Getting started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+```
+$ git clone git@github.com:Financial-Times/kmt-header.git
+$ npm install
+$ bower install
+```
+
+Then you can run `$ npm start` and go to `http://local.ft.com:5000/` in your browser to see the footer module running locally on your machine.
+
+To be able to manage footer main settings you can create an `.env` file in the root directory with the following variables:
```

Comment:
These are not really necessary (maybe just the first two). Might have just been copied from kat-footer.